src/styles/cards.scss

```diff
@@ -0,0 +1,127 @@
+.main__container {
+  margin: 0 auto;
+  width: (4 * $card-width) + (3 * $card-margin) + (2 * $inside-padding);
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 2px;
```

Comment:
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)